pom.xml

```diff
@@ -35,6 +35,7 @@
         <version.commons.lang3>3.9</version.commons.lang3>
         <version.fasterxml>2.9.9</version.fasterxml>
         <version.github.mmazi>2.0.2</version.github.mmazi>
+        <version.resilience4j>0.17.0</version.resilience4j>
```

Comment:
Current stable version is 1.0.0. Is it any reason why you decided to use 0.17.0?

Reply:
1.0.0 was released a few days ago, I'll upgrade